content/docs/04-clusters/06-cluster-management/04-os-patching.md

```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 
 
-
-
 # Overview
 
 Palette deploys Kubernetes clusters using pre-built VM images. The operating system on these images is the latest patch version at the time of building the image for the supported major-minor streams. As an example, if Ubuntu 18.04 is selected for the OS layer during provisioning, the OS on the cluster nodes might be 18.04.3 LTE, assuming that was the latest at the time the VM image was built. However, newer versions continue to be published in the upstream repositories as improvements, bug fixes and security patches are released. OS Patching allows operating system on the running clusters nodes to be updated to the latest patch version so that it is up-to-date with the latest fixes. In our example, let us assume 18.04.4 LTE and 18.04.5 LTE are released over time to fix important security issues. OS Patching operation will identify 18.04.5 as the latest version and upgrade to it on the cluster nodes. Following choices are available for patching the operating system to the latest version.
  
-
 # Patch on boot
 During the cluster creation, while configuring the cluster (Cluster Configuration), the user can select “Patch OS on boot”. In this case, the operating system on all cluster nodes will be updated to the latest when the cluster VMs are initially deployed.
 
 |  **During Cluster Creation**       |
 | -----------------------------------|
-|Cluster Configuration -> "Patch on Boot" |
+|Cluster Configuration -> **Patch on Boot**|
+
+
+# Reboot If Required

```

Comment:
```suggestion
# Reboot if Required

```
This is one of the things that is annoying with title case headings, but the norm is to not title case words fewer than three words, such as the, as, in, to, if, and, and many more.